sdks/java/harness/src/test/java/org/apache/beam/fn/harness/CachesTest.java

```diff
@@ -160,4 +160,20 @@ private void testCache(Cache<String, String> cache) {
     assertEquals("value2", cache.computeIfAbsent("key2", (unused) -> "value2"));
     assertEquals("value2", cache.peek("key2"));
   }
+
+  @Test
+  public void testDescribeStats() throws Exception {
+    Cache<Integer, WeightedValue<String>> cache = Caches.forMaximumBytes(1000 * 1048576L);
+    for (int i = 0; i < 100; ++i) {
+      cache.computeIfAbsent(i, (key) -> WeightedValue.of("value", 1048576L));
+      cache.peek(i);
+      cache.put(100 + i, WeightedValue.of("value", 1048576L));
+    }
+
+    assertThat(cache.describeStats(), containsString("used/max 200/1000 MB"));
```

Comment:
would be good to have some test of the correctness of the used bytes as item weight is updated or evictions occur

Reply:
Added eviction count and checked evictions and shrinkage.